neuromation/cli/admin.py

```diff
@@ -26,4 +28,47 @@ def admin() -> None:
     )
 
 
+@command()
+@click.argument("cluster_name", required=True, type=str)
+@click.argument("user_name", required=True, type=str)
+@click.argument(
+    "role",
+    required=False,
+    default="user",
+    metavar="ROLE",
+    type=click.Choice(list(_ClusterUserRoleType)),
+)
+@async_cmd()
+async def add_cluster_user(
+    root: Root, cluster_name: str, user_name: str, role: str
+) -> None:
+    """
+    Add user access to specified cluster with one of 3 roles: admin, manager or user
+    """
+    await root.client._admin.add_cluster_user(cluster_name, user_name, role)
+    if not root.quiet:
+        click.echo(
+            f"Added {click.style(user_name, bold=True)} to cluster "
```

Comment:
Maybe use attributes of the result of `add_cluster_user()`: `user.user_name`, `user.role`? Just for the case if the server changes them.